benchmarks/bge_m3/notebooks/plot_onnx_cpu_fp32.ipynb

```
@@ -0,0 +1,310 @@
+    "    records = get_filtered(lang)\n",
+    "    clear_output(wait=True)\n",
+    "    display(lang_dropdown)\n",
+    "    print(f\"\\nShowing {len(records)} runs for language='{lang}'  \"\n",
+    "          f\"(batch_sizes={sorted(records['batch_size'].unique())}, \"\n",
+    "          f\"max_lengths={sorted(records['max_length'].unique())})\")\n",
+    "    display(records[[\"run\",\"language\",\"batch_size\",\"max_length\",\n",
+    "                    \"embedding_tokens_per_sec\",\"end_to_end_tokens_per_sec\",\"bottleneck_type\"]])\n",
+    "\n",
+    "lang_dropdown.observe(on_lang_change, names=\"value\")\n",
+    "display(lang_dropdown)\n",
+    "\n",
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)